src/uct/sm/scopy/cma/cma_iface.c

```diff
@@ -70,12 +70,12 @@ static ucs_status_t uct_cma_iface_query(uct_iface_h tl_iface,
 
 static int
 uct_cma_iface_is_reachable(const uct_iface_h tl_iface,
-                           const uct_device_addr_t *dev_addr,
-                           const uct_iface_addr_t *tl_iface_addr)
+                           const uct_iface_is_reachable_params_t *params)
```

Comment:
can we inline uct_cma_iface_is_reachable code in uct_cma_iface_is_reachable_v2?

Reply:
fixed